src/library/required/extension/omg-pixel-collector-web-preloader.js

```diff
@@ -117,11 +117,15 @@
         }
     }
 
-    function getCollectorWebResource(messageType, batched, onlyS3) {
+    function getCollectorWebResource(messageType, batched, s3AndKafka) {
         batched = batched || false;
         var base = omg.isProd() ? COLLECTOR_WEB_PROD : COLLECTOR_WEB_TEST;
-        if(onlyS3){
-            return base + '/' + messageType + '.json?stream=false&persist=true&batch=' + batched;
+        if(s3AndKafka){
+            if(omg.isProd()) {
+                return base + '/' + messageType + '.json?stream=false&persist=true&batch=' + batched;
```

Comment:
@tyang-expedia will create config for persist and stream. We always want batch = true